Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -72,6 +77,47 @@ local devices = {
 	SO69 = 64,
 }
 
+function MiG_29A:defineCabinTempSwitch(identifier, device_id, arg_number, category, description)
+	local alloc = self:allocateInt(3)
+	self:addExportHook(function(dev0)
+		local val = MiG_29A.round2(dev0:get_argument_value(arg_number), 2)
+		if val == 0 then
+			alloc:setValue(0)
+		elseif val == 0.15 then
+			alloc:setValue(1)
+		elseif val == 0.45 then
+			alloc:setValue(2)
+		elseif val == 0.75 then
+			alloc:setValue(3)
```

Comment:
These are floats, so there's a good chance this could get out of sync. I'd recommend comparing using some sort of range or epsilon